css/freebusy.scss

```diff
@@ -65,12 +64,11 @@
 	}
 
 	&__colors {
-		display: flex;
-		justify-content: space-between;
-
+		width: 100%;
```

Comment:
```suggestion
		width: 100%;
		display:flex;
		flex-wrap: wrap;
```

This will make it look nice on mobile:

|before|after|
|---|---|
|![image](https://user-images.githubusercontent.com/7427347/196931237-2b117ad8-c940-4930-892d-c70593b30d3c.png)|![image](https://user-images.githubusercontent.com/7427347/196930540-f4904214-e8d1-4aa9-90bc-e9c8113d0a75.png)|




while still implementing your changes :)

Reply:
Good point @miaulalala  ! Thanks (committed)